pyrokinetics/equilibrium/geqdsk.py

```diff
@@ -0,0 +1,128 @@
+from contextlib import redirect_stdout
+
+from .equilibrium import Equilibrium, equilibrium_reader
+from .flux_surface import _flux_surface_contour
+from ..readers import Reader
+from ..typing import PathLike
+from ..units import ureg as units
+
+import numpy as np
+from freegs import _geqdsk
```

Comment:
Might be worth switching over to reading via FreeQDSK which is a bespoke GEQDSK file reader rather than using a GEQDSK reader in a equilibrium solver. I think all the variable names are the same so hopefully its a simple switch


https://github.com/ZedThree/FreeQDSK